pkg/rbac/cache.go

```diff
@@ -25,12 +22,9 @@ type Cache struct {
 	// Clients to external APIs.
 	// Defining these here allow the tests to replace with a mock client.
 	authnClient  authnv1.AuthenticationV1Interface
-	authzClient  authzv1.AuthorizationV1Interface
 	corev1Client corev1.CoreV1Interface
-
-	kubeClient kubernetes.Interface
```

Comment:
Why do we remove the kubeClient from cache?

Reply:
It was not getting used.